packages/@aws-cdk/core/test/private/test.tree-metadata.ts

```diff
@@ -22,41 +18,91 @@ export = {
       tree: {
         id: 'App',
         path: '',
-        children: [
-          {
+        children: {
+          Tree: {
             id: 'Tree',
-            path: 'Tree'
+            path: 'Tree',
```

Comment:
Are these trailing commas stripped out somewhere? I'm not sure if that's compliant with the JSON specification.
I think it is if you're using something like [JSON5](https://json5.org/)

Reply:
It is valid Javascript and ECMAScript - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
awesome, thanks for confirming! (was failing my linter, but looks like my linter validates compliance against pre ES5 as well